examples/OrderEdit/StagedOrderSetLocaleAction.json

```diff
@@ -0,0 +1,4 @@
+{
+  "action": "setLocale",
+  "locale": "de-de"
```

Comment:
Please change to a valid locale e.g. `de-DE`

Reply:
done